liquid.py

```diff
@@ -226,14 +231,33 @@ def nomad_address():
     print(first(members, 'members'))
 
 
+def set_collection_defaults(name, settings):
+    settings['collection_name'] = name
+    settings.setdefault('workers', 2)
+
+
+def set_hcl_paths(hcl):
+    hcl = hcl.replace('__LIQUID_VOLUMES__', liquid_volumes)
+    return hcl.replace('__LIQUID_COLLECTIONS__', liquid_collections)
+
+
+def run_collection_job(name, settings):
+    set_collection_defaults(name, settings)
+
+    with open('collection.nomad') as collection_file:
+        template = Template(collection_file.read())
+
+        hcl = template.substitute(settings)
```

Comment:
We're doing two passes of substitution, one for `__LIQUID_*__` and another for the collection parameters. Why don't we do a single pass and convert all our template variables to the same format, either `__VALUE__` or `$value`, whichever feels more readable (I'm partial towards `__`, just because it's clearly different from what a Nomad template looks like).